### DISCUSSION TITLE
Ethernet on NSPanelPro 86

### DISCUSSION THREAD

**Discussion by Wuehli** [seaky/nspanel_pro_tools_apk#170] [General]
Hi Folks,

i noticed that my NSPanelPro86 does not support wired network connections, or at least it doesn't work with the adapter i tried (from Ugreen), on my NSPanalPro 120 it worked fine with this adapter. I tried a firmware upgrade to 3.5 on the 86 (from 2.2).
Anybody any ideas? Thanks.
cheers

**Comment by bobzer** [comment]
yep also tried with the 86, no success, but didn't try yet with the 120, glad it could works, did you do something in particular to make it works ?

**Comment by seaky** [comment]
What do you see in `dmesg` on np86?

**Comment by evoblicec** [comment]
> Hi Folks,
> 
> i noticed that my NSPanelPro86 does not support wired network connections, or at least it doesn't work with the adapter i tried (from Ugreen), on my NSPanalPro 120 it worked fine with this adapter. I tried a firmware upgrade to 3.5 on the 86 (from 2.2). Anybody any ideas? Thanks. cheers

Hi @Wuehli could you detail how you got the Ethernet adaptor to work on the NSPanel Pro 120?

I’ve USB access but can’t figure out the Ethernet part yet. What dongle did you use?

**Comment by Wuehli** [comment]
Hi,

I tried multiple once, one was a amazon Basic 100mbit usb2.0 with adapter that worked fine. Also a ugreen network adapter for cromecast that had microusb already worked fine too. I simply plugged them in and they worked.

02.08.2025 13:51:05 evoblicec ***@***.***>:

**Reply by evoblicec** [reply]
> Hi, I tried multiple once, one was a amazon Basic 100mbit usb2.0 with adapter that worked fine. Also a ugreen network adapter for cromecast that had microusb already worked fine too. I simply plugged them in and they worked. 02.08.2025 13:51:05 evoblicec ***@***.***>:
> […](#)
> Hi Folks, i noticed that my NSPanelPro86 does not support wired network connections, or at least it doesn't work with the adapter i tried (from Ugreen), on my NSPanalPro 120 it worked fine with this adapter. I tried a firmware upgrade to 3.5 on the 86 (from 2.2). Anybody any ideas? Thanks. cheers Hi @Wuehli[https://github.com/Wuehli] could you detail how you got the Ethernet adaptor to work on the NSPanel Pro 120? I’ve USB access but can’t figure out the Ethernet part yet. What dongle did you use? — Reply to this email directly, view it on GitHub[[#170 (comment)](https://github.com/seaky/nspanel_pro_tools_apk/discussions/170#discussioncomment-13969519)], or unsubscribe[https://github.com/notifications/unsubscribe-auth/AVENSKVDKOZ5PJIAOBTWYC33LSQ2PAVCNFSM6AAAAABULEBF32VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGOJWHE2TCOI]. You are receiving this because you were mentioned. [Verfolgungsbild][https://github.com/notifications/beacon/AVENSKT45DYIIG6JID2H6AD3LSQ2PA5CNFSM6AAAAABULEBF32WGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQA2UUG6.gif]

Thanks for the quick reply. Could you share the adapteur references?

What have configured in the parameter? Did you get DHCP working out of the box ?

**Reply by evoblicec** [reply]
> Hi,
> 
> 
> 
> I tried multiple once, one was a amazon Basic 100mbit usb2.0 with adapter that worked fine. Also a ugreen network adapter for cromecast that had microusb already worked fine too. I simply plugged them in and they worked.
> 
> 
> 
> 02.08.2025 13:51:05 evoblicec ***@***.***>:
> 
> 
> 
> > 
> 
> > Hi Folks,
> 
> > 
> 
> > i noticed that my NSPanelPro86 does not support wired network connections, or at least it doesn't work with the adapter i tried (from Ugreen), on my NSPanalPro 120 it worked fine with this adapter. I tried a firmware upgrade to 3.5 on the 86 (from 2.2). Anybody any ideas? Thanks. cheers
> 
> > 
> 
> > Hi @Wuehli[https://github.com/Wuehli] could you detail how you got the Ethernet adaptor to work on the NSPanel Pro 120?
> 
> > 
> 
> > I’ve USB access but can’t figure out the Ethernet part yet. What dongle did you use?
> 
> > 
> 
> > —
> 
> > Reply to this email directly, view it on GitHub[https://github.com/seaky/nspanel_pro_tools_apk/discussions/170#discussioncomment-13969519], or unsubscribe[https://github.com/notifications/unsubscribe-auth/AVENSKVDKOZ5PJIAOBTWYC33LSQ2PAVCNFSM6AAAAABULEBF32VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGOJWHE2TCOI].
> 
> > You are receiving this because you were mentioned.
> 
> > [Verfolgungsbild][https://github.com/notifications/beacon/AVENSKT45DYIIG6JID2H6AD3LSQ2PA5CNFSM6AAAAABULEBF32WGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQA2UUG6.gif]
> 
> > 
> 
> 

Hi,

Did you use an OTG dongle? I cannot get anything through USB except power. The USB is also greyed out in the settings.
